packages/app/src/components/home/HomePage.tsx

```diff
@@ -60,6 +60,7 @@ const useCatalogStyles = makeStyles({
     '& img': {
       height: 80,
       width: 80,
+      'objectFit': 'contain',
```

Comment:
nit: quotes are not necessary on the key here, please remove

Reply:
Fixed in the lasest commit.